CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `open`, `defaultOpen` and `onOpenChange` props for `Dropdown` component (controlled mode) @Bugaa92 ([#900](https://github.com/stardust-ui/react/pull/900))
 - Add `accessibility` prop to all components that supports it @layershifter ([#927](https://github.com/stardust-ui/react/pull/927))
 - Export `FocusZone` types @sophieH29 ([#943](https://github.com/stardust-ui/react/pull/943/))
+- Export `call-pstn-full` and `sfb-viewbox` SVG icons to the Teams Theme @thewulf7([#929](https://github.com/stardust-ui/react/pull/968))
```

Comment:
nit: please, lowercase `Theme` -> `theme`

Also, please, ensure that changelog entry is provided under `Unreleased` section after merging with recent `master`.